dev-libs/wlroots/metadata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>speedjack95@gmail.com</email>
+		<name>Niccolò Scatena</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Pluggable, composable, unopinionated modules for building a Wayland
+		compositor.
+
+		wlroots implements a huge variety of Wayland compositor features and
+		implements them right. By using wlroots, you get high performance,
+		excellent hardware compatibility, broad support for many wayland
+		interfaces, and comfortable development tools.
+
+		wlroots is developed under the direction of the <pkg>dev-libs/sway</pkg> project.
+	</longdescription>
+	<use>
+		<flag name="elogind">Enable support for rootless session via elogind</flag>
+		<flag name="icccm">Use xcb-icccm (from <pkg>x11-libs/xcb-util-wm</pkg>) util library for improved Xwayland introspection</flag>
+		<flag name="rootston">Build the rootston example compositor</flag>
+		<flag name="x11-backend">Enable X11 backend</flag>
+		<flag name="xkb">Use xcb-xkb (from <pkg>x11-libs/libxcb</pkg>) util library for improved keyboard handling on the X11 backend</flag>
+		<flag name="X">Enable support for X11 applications (XWayland)</flag>
```

Comment:
Ok, I find the x11-backend vs X flags utterly confusing here. Could you elaborate a bit (in the text) what do they actually mean for the user?

Reply:
Should be clearer now.

Reply:
No, it works even with `+X -x11-backend`